app/presenters/flow_presenter.rb

```diff
@@ -26,7 +26,7 @@ def finished?
     current_node.outcome?
   end
 
-  def collapsed_questions
+  def answered_questions
```

Comment:
Nit: Whilst I agree with this change, it probably didn't need to be apart of this PR. Keeping PRs tightly scoped helps reviewers so they don't become overwhelming big and are easier to understand.

Reply:
Sure - I'll separate PR it.